roles/ceph-mds/tasks/docker/start_docker_mds.yml

```diff
@@ -1,4 +1,52 @@
 ---
+# Use systemd to manage container on Atomic host
+- name: generate systemd unit file
+  become: true
+  template:
+    src: "{{ playbook_dir }}/roles/ceph-mds/templates/ceph-mds.service.j2"
```

Comment:
Made change as requested.

On Fri, Jun 3, 2016 at 1:49 AM, Leseb <notifications@github.com> wrote:

> In roles/ceph-mds/tasks/docker/start_docker_mds.yml
> <https://github.com/ceph/ceph-ansible/pull/840#discussion_r65675146>:
>
> > @@ -1,4 +1,52 @@
> >  ---
> > +# Use systemd to manage container on Atomic host
> > +- name: generate systemd unit file
> > +  become: true
> > +  template:
> > +    src: "{{ playbook_dir }}/roles/ceph-mds/templates/ceph-mds.service.j2"
>
> please use {{ role_path }}/templates/ instead.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph-ansible/pull/840/files/959edf7205069887c1414e0fd4fd3ea094de1172#r65675146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APWGwLR-C_O-vmgPRqu8rJuEDGnEhK_5ks5qH-qegaJpZM4ItD49>
> .
>



-- 
Jim Curtis
Principal Software Engineer
Emerging Technologies
Red Hat Inc.

jim.curtis@redhat.com